audio/ddptools/Portfile

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+
+name            ddptools
+version         1.1
+revision        0
+
+categories      audio
+platforms       {darwin any}
+maintainers     {breun.nl:nils @breun} openmaintainer
+license         none
```

Comment:
this is not a valid license type; I doubt this is recognized by the MacPorts' tools to determine whether something is distributable and such.

Reply:
`port lint` didn’t complain about this, while it did complain about `license unknown`. Where can I find what license types are valid? And what should this be set to for software that is closed source, but free to use?

Reply:
I think we take any SPDX code?

Reply:
There are 27 existing ports with `license` set to `none`, so it does seem to be a valid license type. I also believe that MacPorts only considers software redistributable when a port defines a license that it knows to allow redistribution.

There is no explicit license for `ddptools`. The website only says the software "can be used free of charge". What value for `license` would represent this better than `none`?

Reply:
SPDX identifiers: https://spdx.org/licenses/

The important thing, I think, is to convey that this is not redistributable.

Reply:
I've changed it to `license NoMirror`, which [the MacPorts mirror code checks for](https://github.com/macports/macports-base/blob/master/src/port1.0/portmirror.tcl#L56-L59).

Reply:
Per my understanding, this is the list of valid values for the `license` field:

https://trac.macports.org/wiki/PortfileRecipes#licensekeyword

Unfortunately I don't think it's accepting any possible SPDX identifier.

Reply:
Unfortunate that we don't take recent SPDX stuff.

Regardless, though, maybe "Restrictive" is appropriate?

Reply:
There don’t seem to be any restrictions on the usage of this software. It’s just that the source is not available. I’m not even sure that redistribution is not allowed, but I’m fine with erring on the safe side on this.